ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml

```diff
@@ -69,6 +69,20 @@ tests:
   container:
     from: src
   skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+- as: aws-e2e-ccm
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+  steps:
+    cluster_profile: aws
+    test:
+    - ref: windows-e2e-operator-test
+    workflow: openshift-e2e-aws-ccm-ovn-hybrid
+- as: aws-e2e-ccm-install
```

Comment:
Is this job going to be a temporary addition?
CCM will be running in the other jobs at some point, correct?

Reply:
Yes, that's right. In-tree cloud providers will be removed soon, and CCM will be the only supported mechanism.

Reply:
This test should also be added to the WMCB repo